great_expectations/data_context/data_context.py

```diff
@@ -396,6 +397,32 @@ def _normalize_store_path(self, resource_store):
                 resource_store["base_directory"] = os.path.join(self.root_directory, resource_store["base_directory"])
         return resource_store
 
+    def get_existing_local_data_docs_sites_urls(self):
+        """Get file urls for all built local data docs."""
+        ge_dir = os.path.abspath(self.root_directory)
+        sites = self.get_project_config().get("data_docs_sites")
+
+        existing_sites = []
+
+        for site_name, site in sites.items():
+            store_backend = site.get("store_backend")
+
+            # TODO this should probably use a store api and check for types rather than relying on strings in the config file
+            if store_backend and store_backend.get("class_name") == "FixedLengthTupleFilesystemStoreBackend":
+                base_dir = store_backend.get("base_directory")
+                data_docs_index = os.path.join(ge_dir, base_dir, "index.html")
+
+                if os.path.isfile(data_docs_index):
+                    existing_sites.append("file://" + data_docs_index)
```

Comment:
Rather than this approach (which I agree is too brittle), load the class and check `issubclass`, as in the HtmlSiteStore __init__ method; that way you can use the backend's `convert_key_to_filepath` method (which maybe shouldn't be private if we're intending to do this?) so this will work across more configurations.

Reply:
Oooh, Yes. I like that.